Sources/XiEditor/XiCore.swift

```diff
@@ -109,6 +111,10 @@ final class CoreConnection: XiCore {
         let params = ["destination": destination, "frontend_samples": frontendSamples] as AnyObject
         sendRpcAsync("save_trace", params: params)
     }
+    
+    func toggleTailConfig(enabled: Bool) {
```

Comment:
Could you add an unit test for that?
[`testTracingConfigEnabled`](https://github.com/xi-editor/xi-mac/blob/master/Tests/XiEditorTests/XiCoreTests.swift#L37) should be a clear example how to do it 😄

Reply:
Done.